docs/UserGuide.md

```diff
@@ -3,27 +3,49 @@
 ## Introduction
 
 WerkIt! is an application for managing workout routines,
-optimized for use via a Command Line Interface (CLI).
+optimized for use via a Command Line Interface (CLI). This application is for you if you wish to
+have an application to keep track of your workouts and have a system to store your workout routines.
+
+You will be able to create, view, update and delete the workouts and plans you have created and stored them in the application.
+You will also be able to schedule your workout plans for a week (7 days).
+
+### Terminology 
+Please take note of these few terms. It is explained as below:
+
+| Term     | Description                                                                                                              |
+|----------|--------------------------------------------------------------------------------------------------------------------------|
+| Exercise | A single unit of exercise. <br/>Example: "push up", "pull up", "sit up"                                                  |
+| Workout  | An exercise with a quantity. <br/>Example: "5 push up", "5 pull up", "10 sit up"                                         |
+| Plan     | A set of workouts. <br/>Example: A plan named, "Arms" will contains "5 push ups, 5 pull ups"                             |
```

Comment:
Minor punctuation issue! Perhaps remove the comma near "named" and "Arms"?

Reply:
Noted, will remove it